lib/OpenQA/WebAPI.pm

```diff
@@ -69,6 +70,24 @@ sub startup {
     # set cookie timeout to 48 hours (will be updated on each request)
     $self->app->sessions->default_expiration(48 * 60 * 60);
 
+    # register YAML output type
+    $self->app->types->type(yaml => 'text/yaml;charset=UTF-8');
+    $self->app->renderer->add_handler(
+        yaml => sub {
+            my ($renderer, $c, $output, $options) = @_;
+            $renderer->{type} = 'yaml';
```

Comment:
What does this do?

Reply:
This uses the type defined above, that is `text/yaml` with UTF-8 encoding. Otherwise it defaults to HTML.

Reply:
Oh correction, that was a typo I didn't remove. Nevermind. The real format is below.